source/project-stories/reed-learning.erb

```diff
@@ -164,11 +174,20 @@ related:
       <p class="highlight__large-text"></p>
       <p class="highlight__small-text"><%= current_page.data.what_we_did_4.text %></p>
     </div>
-  </section>
+  </section> -->
```

Comment:
If we're deleteing things, it also makes sense to remove all the `what_we_did_xxx` sections from the frontmatter as they're only referenced in these commented out sections and if they're going...